src/app/active-mode/Views/current-activity/VendorBounties.tsx

```diff
@@ -91,12 +111,55 @@ function VendorBounties({
     }
   });
 
+  if (!bounties.length) {
+    return null;
+  }
+
+  const suggestedBounties = bountiesForActivity(defs, bounties, activity);
+
+  const ownedBounties: VendorItem[] = [];
+  const unownedBounties: VendorItem[] = [];
+  suggestedBounties.forEach((vendorItem) => {
+    vendorItem.item &&
+      (ownedItemHashes.has(vendorItem.item.hash) ? ownedBounties : unownedBounties).push(
+        vendorItem
+      );
+  });
+
+  const ownedBountyGuide = ownedBounties
+    .map(({ item }) => item ?? undefined)
+    .filter((bounties): bounties is DimItem => bounties !== undefined);
```

Comment:
You can also just do `_.compact(ownedBounties.map((i) => i.item))` which will have the right types

Reply:
ah, thats perfect!